DEBUG 2023-10-11 06:36:35.628 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:36:35.628 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:36:35.628 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:36:35.629 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:36:35.630 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-100to200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:36:35.630 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:36:35.634 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:36:35.634 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:36:35.636 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:36:49.865 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:36:49.866 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:36:49.866 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:36:49.866 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:36:49.867 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-2500toInf_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:36:49.867 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:36:49.871 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:36:49.871 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:36:49.872 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:39:07.564 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:39:07.564 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:39:07.564 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:39:07.565 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:39:07.566 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:39:07.566 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:39:07.569 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:39:07.569 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:39:07.570 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:41:12.481 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:41:12.481 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:41:12.481 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:41:12.482 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:41:12.482 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-1200to2500_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:41:12.483 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:41:12.485 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:41:12.486 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:41:12.487 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:41:37.339 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:41:37.340 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:41:37.340 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:41:37.341 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:41:37.341 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-70to100_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:41:37.341 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:41:37.344 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:41:37.345 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:41:37.346 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:42:52.746 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:42:52.746 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:42:52.746 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:42:52.747 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:42:52.748 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:42:52.748 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:42:52.751 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:42:52.752 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:42:52.753 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:44:34.991 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:44:34.991 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:44:34.991 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:44:34.992 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:44:34.993 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-2500toInf_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:44:34.993 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:44:34.996 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:44:34.996 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:44:34.997 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:45:00.378 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:45:00.378 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:45:00.378 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:45:00.379 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:45:00.380 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:45:00.380 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:45:00.384 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:45:00.384 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:45:00.385 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:45:13.884 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:45:13.884 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:45:13.884 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:45:13.885 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:45:13.886 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2017UL.py
DEBUG 2023-10-11 06:45:13.886 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:45:13.889 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:45:13.889 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:45:13.897 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:45:40.217 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:45:40.218 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:45:40.218 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:45:40.219 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:45:40.220 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:45:40.220 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:45:40.224 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:45:40.224 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:45:40.225 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:45:52.356 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:45:52.356 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:45:52.357 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:45:52.357 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:45:52.358 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:45:52.358 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:45:52.361 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:45:52.361 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:45:52.368 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:46:30.954 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:46:30.955 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:46:30.955 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:46:30.956 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:46:30.956 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:46:30.957 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:46:30.960 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:46:30.960 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:46:30.973 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:50:31.817 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:50:31.818 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:50:31.818 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:50:31.819 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:50:31.819 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-1200to2500_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:50:31.819 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:50:31.823 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:50:31.823 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:50:31.830 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:53:15.101 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:53:15.101 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:53:15.101 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:53:15.102 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:53:15.103 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2018UL.py
DEBUG 2023-10-11 06:53:15.103 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:53:15.107 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:53:15.107 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:53:15.120 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:53:52.888 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:53:52.889 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:53:52.889 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:53:52.890 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:53:52.890 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_top_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:53:52.890 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:53:52.894 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:53:52.894 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:53:52.896 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:53:53.175 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:53:53.175 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:53:53.175 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:53:53.175 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:53:53.176 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-800to1200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:53:53.176 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:53:53.179 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:53:53.179 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:53:53.181 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:55:07.456 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:55:07.457 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:55:07.457 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:55:07.458 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:55:07.458 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_top_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:55:07.459 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:55:07.462 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:55:07.462 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:55:07.466 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:55:44.343 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:55:44.343 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:55:44.343 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:55:44.344 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:55:44.344 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:55:44.344 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:55:44.347 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:55:44.348 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:55:44.355 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:56:46.233 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:56:46.234 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:56:46.234 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:56:46.235 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:56:46.235 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-70to100_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 06:56:46.235 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:56:46.239 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:56:46.239 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:56:46.240 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 06:58:54.611 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 06:58:54.611 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 06:58:54.611 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 06:58:54.612 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 06:58:54.613 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-800to1200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-11 06:58:54.613 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 06:58:54.616 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 06:58:54.617 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 06:58:54.627 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 07:02:37.811 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 07:02:37.811 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 07:02:37.811 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 07:02:37.812 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 07:02:37.813 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 07:02:37.813 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 07:02:37.816 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 07:02:37.816 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 07:02:37.818 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-11 07:03:17.496 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-11 07:03:17.497 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-11 07:03:17.497 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-11 07:03:17.498 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-11 07:03:17.498 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-100to200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-11 07:03:17.498 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-11 07:03:17.501 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-11 07:03:17.501 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-11 07:03:17.505 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-24 09:29:14.964 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-24 09:29:14.964 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-24 09:29:14.964 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-24 09:29:14.964 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-24 09:29:14.966 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-100to200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-24 09:29:14.966 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-24 09:29:14.980 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-24 09:29:14.980 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-24 09:29:15.002 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-24 09:36:15.968 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-24 09:36:15.968 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-24 09:36:15.968 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-24 09:36:15.969 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-24 09:36:15.970 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-100to200_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpostVFP.py
DEBUG 2023-10-24 09:36:15.970 UTC: 	 Loading CRAB configuration file.
ERROR 2023-10-24 09:36:16.068 UTC: 	 Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2023-10-24 09:36:16.068 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 94, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter General.requestName should not be longer than 100 characters.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2023-10-24 09:36:16.078 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:17:57.491 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:17:57.491 UTC: 	 Running on: Linux m1.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:17:57.491 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:17:57.492 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 09:17:57.500 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_TTToSemiLeptonic_TuneCP5_13TeV-powheg-pythia8__2018UL.py
DEBUG 2023-10-25 09:17:57.500 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 09:17:57.762 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 09:17:57.765 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_TTToSemiLeptonic_TuneCP5_13TeV-powheg-pythia8__2018UL' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 09:17:57.765 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_TTToSemiLeptonic_TuneCP5_13TeV-powheg-pythia8__2018UL' already exists 
Please change the requestName in the config file
INFO 2023-10-25 09:17:57.804 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:53.565 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:53.566 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:53.566 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:53.567 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:53.575 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:53.576 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:53.704 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:53.966 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:53.967 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:53.967 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:53.967 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:53.968 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:53.968 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:53.970 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:54.259 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:54.260 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:54.260 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:54.260 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:54.261 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:54.261 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:54.262 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:54.539 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:54.540 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:54.540 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:54.540 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:54.540 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:54.541 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:54.542 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:54.810 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:54.810 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:54.810 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:54.810 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:54.811 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:54.811 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:54.813 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:55.078 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:55.078 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:55.078 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:55.078 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:55.079 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:55.079 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:55.080 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:55.349 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:55.349 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:55.349 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:55.349 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:55.350 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:55.350 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:55.351 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:55.619 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:55.620 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:55.620 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:55.620 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:55.620 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:55.621 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:55.622 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:55.886 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:55.886 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:55.886 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:55.887 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:55.887 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:55.887 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:55.889 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:56.155 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:56.155 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:56.156 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:56.156 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:56.156 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:56.156 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:56.158 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:56.420 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:56.421 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:56.421 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:56.421 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:56.421 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:56.422 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:56.423 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:56.690 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:56.690 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:56.690 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:56.690 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:56.691 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:56.691 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:56.693 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:56.962 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:56.963 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:56.963 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:56.963 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:56.964 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:56.964 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:56.965 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:57.244 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:57.244 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:57.244 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:57.244 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:57.245 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:57.245 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:57.247 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:58.015 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:58.015 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:58.015 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:58.016 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:58.016 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:58.016 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:58.018 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:58.275 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:58.275 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:58.275 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:58.276 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:58.276 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:58.276 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:58.278 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:58.533 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:58.534 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:58.534 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:58.534 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:58.535 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:58.535 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:58.536 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:58.793 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:58.793 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:58.793 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:58.793 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:58.794 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:58.794 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:58.795 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:59.056 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:59.057 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:59.057 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:59.057 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:59.057 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:59.058 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:59.059 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:59.316 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:59.317 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:59.317 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:59.317 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:59.318 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:59.318 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:59.319 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:59.572 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:59.572 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:59.572 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:59.572 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:59.573 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:59.573 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:59.575 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 09:56:59.830 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 09:56:59.830 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 09:56:59.830 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 09:56:59.830 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 09:56:59.831 UTC: 	 CRAB configuration file crabConfig.py not found.
ERROR 2023-10-25 09:56:59.831 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 316, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 51, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
INFO 2023-10-25 09:56:59.832 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:00:04.212 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:00:04.212 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:00:04.212 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:00:04.213 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:00:04.214 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-1200to2500_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:00:04.214 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:00:04.252 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:00:04.252 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-1200to2500_TuneCP5_PSweights_13TeV-madgraphMLM-pythia' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:00:04.252 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-1200to2500_TuneCP5_PSweights_13TeV-madgraphMLM-pythia' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:00:04.257 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:00:18.197 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:00:18.198 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:00:18.198 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:00:18.199 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:00:18.199 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:00:18.199 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:00:18.273 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:00:18.273 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythi' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:00:18.273 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-200to400_TuneCP5_PSweights_13TeV-madgraphMLM-pythi' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:00:18.275 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:00:59.855 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:00:59.855 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:00:59.855 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:00:59.856 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:00:59.857 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:00:59.857 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:00:59.905 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:00:59.905 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pyth' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:00:59.906 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-400to600_TuneCP5_PSweights_13TeV-madgraphMLM-pyth' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:00:59.910 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:01:13.396 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:01:13.396 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:01:13.396 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:01:13.397 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:01:13.398 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:01:13.398 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:01:13.458 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:01:13.458 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pyt' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:01:13.458 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-600to800_TuneCP5_PSweights_13TeV-madgraphMLM-pyt' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:01:13.460 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:01:28.820 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:01:28.821 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:01:28.821 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:01:28.822 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:01:28.822 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-70to100_TuneCP5_PSweights_13TeV-madgraphMLM-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:01:28.822 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:01:28.842 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:01:28.842 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-70to100_TuneCP5_PSweights_13TeV-madgraphMLM-pythi' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:01:28.842 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_DYJetsToLL_M-50_HT-70to100_TuneCP5_PSweights_13TeV-madgraphMLM-pythi' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:01:28.844 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:02:14.245 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:02:14.245 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:02:14.246 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:02:14.246 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:02:14.247 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2016ULpreVFP.py
DEBUG 2023-10-25 10:02:14.247 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:02:14.611 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:02:14.611 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:02:14.612 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:02:14.613 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:02:28.021 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:02:28.022 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:02:28.022 UTC: 	 Executing command: 'submit'
DEBUG 2023-10-25 10:02:28.023 UTC: 	 Found CRAB cache file /user/jusong/.crab3
INFO 2023-10-25 10:02:28.023 UTC: 	 Will use CRAB configuration file crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8__2018UL.py
DEBUG 2023-10-25 10:02:28.023 UTC: 	 Loading CRAB configuration file.
DEBUG 2023-10-25 10:02:28.057 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2023-10-25 10:02:28.057 UTC: 	 Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8' already exists 
Please change the requestName in the config file
ERROR 2023-10-25 10:02:28.057 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 320, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/ClientUtilities.py", line 387, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/tasks/crab_TopNanoAODv9-1-1_ST_t-channel_antitop_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8' already exists 
Please change the requestName in the config file
INFO 2023-10-25 10:02:28.061 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
DEBUG 2023-10-25 10:56:52.467 UTC: 	 CRAB Client version: v3.230612
DEBUG 2023-10-25 10:56:52.468 UTC: 	 Running on: Linux m3.iihe.ac.be 3.10.0-1160.92.1.el7.x86_64 #1 SMP Tue Jun 20 11:48:01 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2023-10-25 10:56:52.468 UTC: 	 Executing command: 'resubmit'
DEBUG 2023-10-25 10:56:52.468 UTC: 	 Found CRAB cache file /user/jusong/.crab3
ERROR 2023-10-25 10:56:52.469 UTC: 	 Error: crab_TopNanoAODv9-1-1_ZZZ_TuneCP5_13TeV-amcatnlo-pythia8__2017UL is not a valid CRAB project directory.
ERROR 2023-10-25 10:56:52.469 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 159, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/bin/crab.py", line 144, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/resubmit.py", line 30, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 275, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.230612.00/lib/CRABClient/Commands/SubCommand.py", line 642, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_TopNanoAODv9-1-1_ZZZ_TuneCP5_13TeV-amcatnlo-pythia8__2017UL is not a valid CRAB project directory.
INFO 2023-10-25 10:56:52.474 UTC: 	 Log file is /ada_mnt/ada/user/jusong/analysis/CMSSW_10_6_27/src/PhysicsTools/NanoAOD/test/crab_tasks/crab.log
